operator/v1/types_dns.go

```diff
@@ -73,6 +83,29 @@ type ForwardPlugin struct {
 	Upstreams []string `json:"upstreams"`
 }
 
+// DNSNodePlacement describes the node scheduling configuration for DNS pods.
+type DNSNodePlacement struct {
+	// nodeSelector is the node selector applied to DNS pods.
+	//
+	// If unset, the default is the following:
```

Comment:
indicate that this default may change over time.

Reply:
I don't foresee its changing, but I'll add a comment to leave us the option to do so.